data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaConsumerConfig.java

```diff
@@ -0,0 +1,12 @@
+package org.opensearch.dataprepper.plugins.kafka.configuration;
+
+public interface KafkaConsumerConfig extends KafkaConnection{
+
+    AwsConfig getAwsConfig();
+
+    EncryptionConfig getEncryptionConfig();
```

Comment:
Should this be part of `KafkaConnection` instead? I'd expect that the encryption configuration is related more to how we connect to Kafka than how we consume. Also, wouldn't a producer need it?